### ISSUE TITLE
IntlPolyfill is not defined

### ISSUE THREAD

**Issue by viniciusavieira** [codesandbox/codesandbox-client#2100]
# ❔ Question: Has something been changed that relates to ```intl```

I've created some codesandboxes about 3 months ago, to display examples of usage of intl into some scenarios I needed to explore.
About 15 days ago I noticed that most of them where broken, and I did not update anything.

So I've started digging into it, and found a workaround, but the problem is:
This behavior only happens in codesandbox, if I run the same code on a local project, it runs as expected, but in codesandbox, it fails because a depedencie.

### Link to sandbox: (optional)
Scenario with problem: https://codesandbox.io/s/yq2jz85jx1
Fixed scenario: https://codesandbox.io/s/yq1434443z

Local scenario with problem code running:
![image](https://user-images.githubusercontent.com/2073019/60144414-a163b480-9798-11e9-933e-762114b2b68b.png)

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | PROD-1561452426-ed898ddbb
| Browser          | Chrome Version 75.0.3770.100 (Official Build) (64-bit)
| Operating System | Linux Ubuntu 18.04.2 LTS

**Comment by lbogdan** [comment]
Hey @viniciusavieira ,

What was the workaround? From a quick look at those two sandboxes, it seems to be adding `intl-locales-supported` dependency, did I get that right?

**Comment by viniciusavieira** [comment]
By looking at it now, it seems that the solution is kinda of an environment setup.

Since tests run on node environment, I need to polyfill some locales when running it, but on codesandbox, I think they run on browser environment correct? 
If this is true, then the browser may have the locales support already, so when I try to add the polyfill I may be attempting to override something and it may be the reason to break the import.

The case is that in the workaround I don't override the locales with ```IntlPolyfill``` sine when checking for ```!areIntlLocalesSupported(["en", "pt", "es"])``` does not pass, it simply does nothing on the setup file.

But for this to work on local machine running Jest, I need to add the ```IntlPolyfill``` that may be the case why it breaks on codesandbox but works on local machine.

If this makes sense to you, it may not be a bug, but just a something that behaves according, since codesandbox is on browser and Jest runs locally on node.

What do you think @lbogdan ?

**Comment by lbogdan** [comment]
I'm not really familiar with native vs. polyfill `intl`, but you're correct when assuming we run all tests inside the browser. 🙂 So I think a conditional polyfill would work in both environments?

**Comment by viniciusavieira** [comment]
Yes, probably this is a setup related problem, and is related to running tests only, since both cases the view presents as expected.

Then only think that really gets my attention is that, 3 or 4 months ago, this broken setup did work as expected, that's why I opened  the issue, and because of it, most of the sandboxes that I have created to test some cases isolated have broken, as they were working fine.
Maybe some codesandbox update has a breaking change that may cause it?
